packages/nextjs/next.config.js

```diff
@@ -9,6 +9,9 @@ const nextConfig = {
   eslint: {
     ignoreDuringBuilds: process.env.NEXT_PUBLIC_IGNORE_BUILD_ERROR === "true",
   },
+  images: {
+    domains: ["avatars.githubusercontent.com"],
+  },
```

Comment:
Using Next `Image` component is really nice practice, but I think everyone in this batch followed using directly HTML `img` tag just to avoid adding remote patter in `next.config.js` and ignoring eslint warning by adding eslint ignore comment checkout https://github.com/BuidlGuidl/batch4.buidlguidl.com/pull/14#issuecomment-2045236597

Maybe let's remove this and use img too ? 🙌

Reply:
My bad I think we already have this and was merged at https://github.com/BuidlGuidl/batch4.buidlguidl.com/pull/22 

Lets resolve merge conflict for this file